lib/client/session.go

```diff
@@ -200,6 +203,22 @@ func (ns *NodeSession) createServerSession() (*ssh.Session, error) {
 	return sess, nil
 }
 
+// selectKeyAgent picks the appropriate key agent for forwarding to the
+// server, if any.
+func selectKeyAgent(tc *TeleportClient) agent.Agent {
+	switch tc.ForwardAgent {
+	case ForwardAgentYes:
+		log.Debugf("Selecting System Key Agent")
```

Comment:
Nit: (derived from https://github.com/golang/go/wiki/CodeReviewComments#error-strings). Let's make our log lines proper english sentences, so:

`Selecting system key agent.`

Reply:
Saw this after hitting Merge, will fix in new PR